lib/drivers/hive.js

```diff
@@ -0,0 +1,72 @@
+const Driver = require('../models/driver');
+const Supply = require('../models/supply');
+
+/**
+ * Hivedb driver. Supports total and circulating supply
+ * for native token HIVE on HIVE blockchain.
+ *
+ * @memberof Driver
+ * @augments Driver
+ */
+class Hive extends Driver {
+  constructor(options) {
+    super(
+      {
+        timeout: 100, // 10 requests per second
+        supports: {
+          circulating: true,
+          blockchains: ['Hive'],
+        },
+      },
+      options,
+    );
+  }
+
+  /** get total supply for native token
+   *
+   * @augments Driver.fetchTotalSupply
+   * @async
+   */
+  async fetchTotalSupply() {
+    const data = await this.request('https://hive-db.com/api/props');
+    const record = data.find((item) => item._id && item._id.$oid === '5f274a0d22e727eccbf74d68');
```

Comment:
I think these props are like blockchain blocks, so it should use the latest block to get the correct supply